storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/operations/get/GetNumberOfResultsStub.java

```diff
@@ -0,0 +1,100 @@
+package com.pushtorefresh.storio.contentresolver.operations.get;
+
+import android.database.Cursor;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.contentresolver.Changes;
+import com.pushtorefresh.storio.contentresolver.StorIOContentResolver;
+import com.pushtorefresh.storio.contentresolver.queries.Query;
+import com.pushtorefresh.storio.test.ObservableBehaviorChecker;
+
+import rx.Observable;
+import rx.functions.Action1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+public class GetNumberOfResultsStub {
+    @NonNull
+    final StorIOContentResolver storIOContentResolver;
+
+    @NonNull
+    private final StorIOContentResolver.Internal internal;
+
+    @NonNull
+    final Query query;
+
+    @NonNull
+    final GetResolver<Integer> getResolverForNumberOfResults;
+
+    @NonNull
+    private final Cursor cursor;
+
+    @NonNull
+    private final Integer numberOfResults;
+
+    @SuppressWarnings("unchecked")
```

Comment:
Can you please remove this `@SuppressWarnings` and add suppressing via comments on required lines? (Alt + Enter and then select suppress by comment, etc)